### DISCUSSION TITLE
Search silences for last 21d

### DISCUSSION THREAD

**Discussion by freeseacher** [prymitive/karma#5867] [General]
Hello there,

I would like to request a new feature for GitHub. During on-call transfers, we would greatly appreciate the ability to transfer new silences. In order to do so, we need the ability to search for silences with an "@age<21d" filter. Currently, this feature is not available and we hope that it can be implemented in the near future.

Thank you for your time and consideration.

**Comment by filippog** [comment]
I'm not the maintainer, however I believe you should extend Alertmanager's data retention via its `--data.retention` command line flag. HTH!

**Comment by freeseacher** [comment]
data retention is not the case. 
my question is about search for new silences
